docs/Getting_Started.mdx

```diff
@@ -72,7 +72,7 @@ The [`config`](./packages/TypeScript_ESLint.mdx) helper sets three parts of ESLi
 
 - [Parser](https://eslint.org/docs/latest/use/configure/parser): set to [`@typescript-eslint/parser`](./packages/Parser.mdx) so ESLint knows how to parse the new pieces of TypeScript syntax.
 - [Plugins](https://eslint.org/docs/latest/use/configure/plugins): set to [`@typescript-eslint/eslint-plugin`](./packages/ESLint_Plugin.mdx) to load [rules listed in our _Rules_ page](./Rules).
-- [Rules](https://eslint.org/docs/latest/use/configure/rules): extends from our [recommended configuration](http://localhost:3000/linting/configs#recommended) to enable our most commonly useful lint rules.
+- [Rules](https://eslint.org/docs/latest/use/configure/rules): extends from our [recommended configuration](https://typescript-eslint.io/linting/configs/#recommended) to enable our most commonly useful lint rules.
```

Comment:
```suggestion
- [Rules](https://eslint.org/docs/latest/use/configure/rules): extends from our [recommended configuration](./linting/configs/#recommended) to enable our most commonly useful lint rules.
```

Let's update it to a relative path? This is a more flexible solution than a hard-coded URL. It will work correctly regardless of the domain name! For example, in Netlify Preview Deployments ([preview deployment for this PR](https://deploy-preview-8463--typescript-eslint.netlify.app/getting-started/#details)). What do you think?